pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -646,7 +646,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			},
 			Rules: []authorizationapi.PolicyRule{
 				authorizationapi.NewRule(readWrite...).Groups(imageGroup).Resources("imagestreamimages", "imagestreammappings", "imagestreams", "imagestreams/secrets", "imagestreamtags").RuleOrDie(),
-				authorizationapi.NewRule("create").Groups(imageGroup).Resources("imagestreamimports").RuleOrDie(),
+				authorizationapi.NewRule("create").Groups(imageGroup).Resources("imagestreamimports", "serviceaccounts").RuleOrDie(),
```

Comment:
> Should be readWrite… I think

Big picture.  That's the wrong API group.  You want `kapiGroup` for "serviceaccounts"